hi-ml/testhiml/testhiml/test_eval_runner.py

```diff
@@ -0,0 +1,64 @@
+from pathlib import Path
+import sys
+from unittest.mock import patch
+
+import pytest
+
+from health_ml import TrainingRunner
+from health_ml.configs.hello_world import (
+    TEST_MAE_FILE,
+    TEST_MSE_FILE,
+    HelloWorld,
+    HelloWorldDataModule,
+)
+from health_ml.eval_runner import EvalRunner
+from health_ml.experiment_config import ExperimentConfig, RunnerMode
+from health_ml.runner import Runner
+from health_ml.utils.checkpoint_utils import CheckpointParser
+from testhiml.test_run_ml import ml_runner_with_container
+
+
+@pytest.fixture(scope="function")
+def hello_world_checkpoint(ml_runner_with_container: TrainingRunner, tmp_path: Path) -> Path:
+    container = ml_runner_with_container.container
+    container.set_output_to(tmp_path)
+    container.max_epochs = 5
+    ml_runner_with_container.run()
+    # Read out the trained checkpoint and use the runner in eval mode
+    return container.get_checkpoint_to_test()
```

Comment:
You can also use the checkpoint in 
hi-ml/hi-ml/testhiml/test_data/hello_world_checkpoint.ckpt 
it will be faster than retraining